CMakeLists.txt

```diff
@@ -91,6 +91,7 @@ if(BUILD_OMNI)
       && CC=${CMAKE_C_COMPILER} CXX=${CMAKE_CXX_COMPILER}
       FC=${CMAKE_Fortran_COMPILER} JAVAC=${Java_JAVAC_EXECUTABLE}
       ${OMNI_COMPILER_SUBMODULE}/configure
+      CFLAGS=${CMAKE_C_FLAGS}
```

Comment:
Can you move it before `configure` just to be constituent with other definition like `JAVAC`/`FC` ...